examples/webgl_multiple_scenes_comparison.html

```diff
@@ -7,6 +7,10 @@
 		<link type="text/css" rel="stylesheet" href="main.css">
 		<style>
 
+			html, body {
+				touch-action: none;
```

Comment:
I was not able to fix touch controls without this CSS, see https://stackoverflow.com/questions/48124372/pointermove-event-not-working-with-touch-why-not

Reply:
I also battled with this today. Safari/iOS doesn't support this.

I ended up doing this instead:

```js
body.addEventListener( 'touchstart', function ( event ) {

    event.preventDefault(); // prevent scrolling

} );
```

Reply:
If I add this code, I get the following runtime error in Chrome:

> webgl_multiple_scenes_comparison:190 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/features/5093566007214080

It only seems to work if I mark the listener as non-passive:
```js
window.addEventListener( 'touchstart', function ( event ) {

    event.preventDefault(); // prevent scrolling

}, { passive: false } );
```
Not sure if this is appropriate...

Reply:
Ah, maybe `document.body` instead?

This `touchstart` and `touch-action` gotcha is such a pain...

Maybe in this case `touch-action` is better actually.

Reply:
> Ah, maybe document.body instead?

Unfortunately, I see the same error with `document.body`.

Reply:
>  Safari/iOS doesn't support this.

Doesn't this limitation prevent a complete migration of the controls like `OrbitControls` to `PointerEvents`? According to the stackoverflow thread it seems that `pointermove` on touch devices will only work if you set `touch-action` to `none`.

Reply:
In `TransformControls` I did this:

```js
domElement.addEventListener( 'touchstart', function ( event ) {

    event.preventDefault(); // prevent scrolling

} );
```

Reply:
Um, why does this approach work with `TransformControls` but not in the example 🤔 ?

Reply:
Maybe because I'm adding it to the `renderer.domElement` directly?

Reply:
I've tested this right now. The errors go away but the UI behavior is still the same. Meaning the `pointermove` gets canceled after a short usage.

It only works if I add all event listeners to `renderer.domElement`. But then it's not possible to just drag the `slider` element but the entire overlay.

Reply:
Okay, I'll merge this and play around with the code 👍